### DISCUSSION TITLE
Is there any way to re-order cards when some of them are lovelace?

### DISCUSSION THREAD

**Discussion by douginoz** [dwainscheeren/dwains-lovelace-dashboard#713] [Q&A]
I've got an auto-populated room where I've hidden a few and left a few.   They all have the 4-arrow mover icon to allow me to move them around.  But I've also added a couple of  non-DD cards (standard Lovelace ones) but those don't seem to have any way to change where they're placed.  I can change the position of a DD card relative to other DD cards, but can't for example move it to above a non-DD card.  I've tried on my desktop (firefox) and iOS app.  The non-DD cards always appear at the top of the room and can't be moved to below the DD cards.

I need to change the order of what's displayed but can't figure out how to do that.  Is there any way?

**Comment by MuppetRus** [comment]
I have the same issue and I guess its because you are using non-DD cards like me and I think that is your answer that DD only supports  DD cards and not non-cards 

Let me know if anybody answers your question but I doubt it

**Reply by douginoz** [reply]
Yes, I think that's the problem. Which is frustrating both because it makes the interface cumbersome and because there doesn't seem to be any interest in addressing this. 

In fact, in the months since I posted the problem, I haven't seen any significant advances in DD to make me think that there's going to be any further development. So I'm looking for alternatives now. As far as I can see, DD is dead apart from the occasional minor fix.

**Comment by dwainscheeren** [comment]
Hi,

I am closing the old Dwains Dashboard 3.x discussions as part of the DD3 cleanup.

DD3 is now in maintenance/community mode and is slowly being phased out. Active development is focused on Dwains Dashboard Next, the newest dashboard project:

https://github.com/dwainscheeren/dwains-dashboard-next

For chat, community help and discussion around DD3/DD Next, please join the Discord:

https://discord.gg/7yt64uX

If you still have a reproducible DD3 bug on the latest 3.9.x release, please open a new issue with current details. For new feature ideas, please try Dwains Dashboard Next.
